### PR TITLE
PEP8 skymatch

### PR DESCRIPTION
**Pull request by mcara**
This PR simply addresses PEP8 issues and also comments out `_calc_sky_orig` function (I just do not have time at this moment to test whether this code can be safely restored. I will do this later but I want to keep it here for convenience.)

I expect no changes to any tests. Most PEP8 issues were line length.

**Checklist for maintainers**
- [ ] added entry in `CHANGES.rst` within the relevant release section
- [ ] updated or added relevant tests
- [ ] updated relevant documentation
- [x] added relevant milestone
- [x] added relevant label(s)
- [x] ran regression tests, post a link to the Jenkins job below.
      [How to run regression tests on a PR](https://github.com/spacetelescope/jwst/wiki/Running-Regression-Tests-Against-PR-Branches)
- [ ] Make sure the JIRA ticket is [resolved properly](https://github.com/spacetelescope/jwst/wiki/How-to-resolve-JIRA-issues)

Regression test is running here: https://plwishmaster.stsci.edu:8081/job/RT/job/JWST-Developers-Pull-Requests/635/

### CODE REVIEW

File: jwst/skymatch/skymatch.py [hunk 1]
```diff
@@ -456,8 +458,14 @@ def _apply_sky(images, sky_deltas, do_global, do_skysub, show_old):
 #     for i in range(ns):
```
Comment (perrygreenfield) [inline jwst/skymatch/skymatch.py:458]: Again, why is the commented code left in?

**Review by perrygreenfield — COMMENTED** [review]
This space intentionally left blank.

File: jwst/skymatch/skymatch.py (hunk 1 above)
Comment (perrygreenfield) [reply jwst/skymatch/skymatch.py:458]: Ok, I see it addressed in your original comment, but still, just keep the code around locally until you are satisfied?

File: jwst/skymatch/skyimage.py [hunk 2]
```diff
@@ -647,107 +647,108 @@ def calc_sky(self, overlap=None, delta=True):
 
         return skyval, npix, polyarea
```
[the remark this answers is not in the document]
Comment (mcara) [reply jwst/skymatch/skyimage.py:649]: No. This was explained in the PR comment: I plan to revert to the original code (commented out in this PR).

**Review by nden — APPROVED** [review]

**Pull request merged**